src/perl/dmtest/dmtests.suites

```diff
@@ -0,0 +1,94 @@
+# -*-cperl-*-
+# vim:filetype=perl
+# $Id$
+#
+
+###########################################################################
+# These suites define convenient groups of tests
+##
+
+# Basic VDO functional tests using filesystems
+my $CCA = "--clientClass=ALBIREO-PMI\\,RAWHIDE";
+my $CCV = "--clientClass=VDO-PMI\\,RAWHIDE";
+
+$aliasNames{Checkin} = "Grouped   --dmtestName=vdo*";
+$aliasNames{Full}    = "Separated --dmtestName=*";
+$aliasNames{Jenkins} = "Grouped   --dmtestName=vdo*";
+$aliasNames{HDD}     = "Separated --dmtestName=vdo* $CCA";
```

Comment:
Isn't dmtestName taking a regular expression to match a substring? The star at the end is both wrong (dot-star if you want to mean an arbitrary string) and unneeded ("vdo" will match anything containing "vdo"; "vdo*" matches "vd" followed by any number of o's).

Reply:
updated